pkg/config/analysis/analyzers/testdata/virtualservice_overlappingmatches.yaml

```diff
@@ -28,10 +28,10 @@ spec:
   http:
   - name: "send product to sample.foo"
     match:
-    - uri:
-        prefix: "/"
     - uri:
         exact: "/"
+    - uri:
```

Comment:
This change seems irrelevant to me. I think without this change, the analyzer test can still pass because the validation analyzers are actually skipped in the test.